srcpkgs/gajim-omemo/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'gajim-omemo'
+pkgname=gajim-omemo
+version=2.6.29
+revision=1
+wrksrc="omemo"
+hostmakedepends="python3-devel unzip"
+makedepends="python3-setuptools"
```

Comment:
and `python3-setuptools` here? It seems the files are just copied.

Reply:
Right. Should not be required.

Reply:
I just copied those from https://dev.gajim.org/gajim/gajim-plugins/wikis/OmemoGajimPlugin and never thought about it.

Reply:
It indeed uses code from setuptools but since it is required at runtime, the dependency should be in `depends`, not `makedepends`.